iroh/src/rpc_protocol.rs

```diff
@@ -177,6 +208,41 @@ impl ServerStreamingMsg<RpcService> for BlobValidateRequest {
     type Response = ValidateProgress;
 }
 
+/// Get the status of a blob
```

Comment:
this is mostly unrelated to the PR. Just a way to check if you have a thing without downloading it all via the rpc.